### DISCUSSION TITLE
Duda ocupar código externo

### DISCUSSION THREAD

**Discussion by TomasKetterer** [IIC2133-PUC/2024-1#86] [Tarea 1] [answered]
En la parte dos, para implementar el árbol encontré código en una página que me sirve para hacer la implementación e incluso para hacer búsquedas. Quería saber si se puede usar código de páginas X o no se puede?

**Comment by sebacarrascop** [comment, marked as answer]
holaa, si puedes usar código externo, recuerda citarlo tanto en él código como en el informe. En esta ocasión no les pedimos readme, pero también podrías crear uno para añadir las citas ahí. :)
